.husky/check-node-version.sh

```
@@ -0,0 +1,37 @@
+# Enforce Node version per .nvmrc - defends against stale global
```

Shellcheck (0.11.0)

[error] Tips depend on target shell and yours is unknown. Add a shebang or a 'shell' directive.

(SC2148)